spec/spec_helper.rb

```diff
@@ -33,12 +33,7 @@
 
   # rspec-mocks config goes here. You can use an alternate test double
   # library (such as bogus or mocha) by changing the `mock_with` option here.
-  config.mock_with :rspec do |mocks|
-    # Prevents you from mocking or stubbing a method that does not exist on
```

Comment:
we should keep this so we can use the allow_any_instance of method with RSpec. what else are we using mocha for?